doc/doxygen/src/deprecated-modules.md

```diff
@@ -0,0 +1,21 @@
+Deprecated Modules    {#modules-deprecated}
+==================
+
+@note   Please keep this in sync with `makefiles/deprecated_modules.inc.mk`
+
+Module                  | Comment                                                       | Scheduled Removal
+----------------------- | ------------------------------------------------------------- | -----------------
+`event_thread_lowest`   | Use `event_thread` instead                                    | 2021.10
+`gnrc_netdev_default`   | Use `netdef_default` instead                                  | 2022.07
+`gnrc_netif_cmd_lora`   | Use `shell_cmd_gnrc_netif_lorawan` instead                    | 2023.07
+`gnrc_pktbuf_cmd`       | Use `shell_cmd_grnc_udp` instead                              | 2023.07
+`heap_cmd`              | Use `shell_cmd_heap` instead                                  | 2023.07
+`i2c_scan`              | Use `shell_cmd_i2c_scan` instead                              | 2023.07
+`md5sum`                | Use `shell_cmd_md5sum` instead                                | 2023.07
+`nice`                  | Use `shell_cmd_nice` instead                                  | 2023.07
+`random_cmd`            | Use `shell_cmd_random` instead                                | 2023.07
+`sema_deprecated`       | Replace `sema_wait_timed()` with `sema_wait_timed_ztimer()`   | 2021.07
+`sha1sum`               | Use `shell_cmd_sha1sum` instead                               | 2023.07
+`sha256sum`             | Use `shell_cmd_sha256sum` instead                             | 2023.07
+`shell_commands`        | Use `shell_cmds_default` instead                              | 2023.07
+`ztimer_now64`          | Use `ztimer64` instead                                        | 2023.04
```

Comment:
The scheduled removal date for `ztimer_now64` is also not in line with what is documented currently: https://doc.riot-os.org/group__sys__ztimer__now64.html. I am working on an alternative.